samples/unit-tests/series-pie/label-align/demo.js

```diff
@@ -1,4 +1,4 @@
-QUnit.test("Change of label alignment after add", function (assert) {
+QUnit.test("Change of label alignment after add (#4652)", function (assert) {
```

Comment:
This isn't a pie series test. Should be moved to the exisiting `unit-tests/svgrenderer/label` tests.